include/upipe-modules/upipe_block_to_sound.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2017 Open Broadcast Systems Ltd
+ *
+ * Authors: Judah Rand
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject
+ * to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+ * CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/** @file
+ * @short Upipe block_to_sound module - converts incoming block urefs to outgoing sound urefs
+ */
+
+#ifndef _UPIPE_MODULES_UPIPE_BLOCK_TO_SOUND_H_
+/** @hidden */
+#define _UPIPE_MODULES_UPIPE_BLOCK_TO_SOUND_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <upipe/upipe.h>
+
+/** signature for pipe **/
```

Comment:
Are you sure ? This comment add a trailing star in the generated documentation.
May I suggest: `/** @This is the signature of the block to sound pipe. */` ?